grunt-ngdocs-index.tmpl

```diff
@@ -31,7 +31,7 @@
       }
 
       addTag('base', {href: baseUrl});
-      addTag('link', {rel: 'stylesheet', href: 'css/prettify.css', type: 'text/css'});
+      // addTag('link', {rel: 'stylesheet', href: 'css/prettify.css', type: 'text/css'});
```

Comment:
Yes, sorry I squashed a commit that explained it. The css/prettify.css changes some sizing causing the filter components to not line up as they would if just using patternfly.